src/components/Employee/Compensation/Edit.tsx

```diff
@@ -90,14 +94,11 @@ export const Edit = () => {
           errorMessage={t('validations.exemptThreshold', {
             limit: format(FLSA_OVERTIME_SALARY_LIMIT),
           })}
-          items={classificationOptions}
+          options={classificationOptions.map(o => ({ value: o.id, label: o.name }))}
```

Comment:
Looks like we already define `classificationOptions` with a map operation elsewhere in this file. Maybe worth just updating that to use value/label instead so we don't have to map twice?

Reply:
Good catch!